scripts/asset_processor/video_asset_processor.py

```diff
@@ -200,19 +205,13 @@ def capture_to_array(self, capture):
             width = frame_data.frame.shape[0]
             pixels += height * width
 
+            if not self.markup_master_frames and self.image_pair_callback is not None:
+                self.image_pair_callback(self.master_samples_hd[i], frame_data.frame, len(frame_list), ts_diff, self.original_path, capture.filename)
```

Comment:
What is the purpose of the image pair callback?

Reply:
Mainly for debug purposes, to visualize or save pairs of matched frames. May also be useful for running some custom verification logic.